### ISSUE TITLE
Fix RSS Error for Google News (Site Search)

### ISSUE THREAD

**Issue by slavakurilyak** [automatisch/automatisch#838]
automatisch v0.3.0 throws the following error when adding Google News RSS:

`parsedData.rss.channel.item is not iterable`

Here's the RSS feed for 'chatgpt' using site search on Google News:

https://news.google.com/rss/search?q=site%3Adocs.google.com%2Fspreadsheets%20chatgpt&hl=en-US&gl=US&ceid=US%3Aen

Here's 'chatgpt' using site search on Google News:

https://news.google.com/search?q=site%3Adocs.google.com%2Fspreadsheets%20chatgpt&hl=en-US&gl=US&ceid=US%3Aen

<img width="863" alt="Untitled" src="https://user-images.githubusercontent.com/6625584/211667594-696366e4-252b-47b2-897d-9060f6a4a823.png">

<img width="1281" alt="screenshot" src="https://user-images.githubusercontent.com/6625584/211665523-1c09ee04-034b-43bd-8d2f-50352336e838.png">

Here's the error log in Chrome:

```
index.ts:71 Uncaught (in promise) Error: {"error":"parsedData.rss.channel.item is not iterable"}
    at new t (index.ts:71:5)
    at QueryManager.ts:246:15
    at n (asyncMap.ts:30:30)
    at asyncMap.ts:19:47
    at new Promise (<anonymous>)
    at Object.then (asyncMap.ts:19:16)
    at Object.next (asyncMap.ts:31:39)
    at Xo (module.js:132:18)
    at Jo (module.js:176:3)
    at t.next (module.js:225:5)
t @ index.ts:71
(anonymous) @ QueryManager.ts:246
n @ asyncMap.ts:30
(anonymous) @ asyncMap.ts:19
then @ asyncMap.ts:19
(anonymous) @ asyncMap.ts:31
Xo @ module.js:132
Jo @ module.js:176
t.next @ module.js:225
(anonymous) @ iteration.ts:13
uu @ iteration.ts:13
next @ Concast.ts:172
Xo @ module.js:132
Jo @ module.js:176
t.next @ module.js:225
next @ index.ts:54
Xo @ module.js:132
Jo @ module.js:176
t.next @ module.js:225
(anonymous) @ createHttpLink.ts:164
Promise.catch (async)
(anonymous) @ useMutation.ts:107
(anonymous) @ index.tsx:86
We @ react-dom.production.min.js:52
Qe @ react-dom.production.min.js:52
(anonymous) @ react-dom.production.min.js:53
Ar @ react-dom.production.min.js:100
Fr @ react-dom.production.min.js:101
(anonymous) @ react-dom.production.min.js:113
Be @ react-dom.production.min.js:292
(anonymous) @ react-dom.production.min.js:50
Rr @ react-dom.production.min.js:105
Jt @ react-dom.production.min.js:75
Xt @ react-dom.production.min.js:74
t.unstable_runWithPriority @ scheduler.production.min.js:18
$o @ react-dom.production.min.js:122
je @ react-dom.production.min.js:292
Kt @ react-dom.production.min.js:73
```

I've noticed that this is happening when the RSS feed is currently empty, but new items are expected to be added in the future. It would be great if automatisch could handle this scenario more gracefully, for example by allowing users to set up "currently empty" RSS feeds with the expectation that new items will be added later.

**Comment by barinali** [comment]
@slavakurilyak I expect this will be covered by #837. However, no variable suggestions would be provided in the flow editor as there are no items in the feed at the moment of the test. Therefore, no variables could be used in the following steps.

**Comment by barinali** [comment]
@slavakurilyak, we have just released version 0.4.0. Could you please test this problem with that version and let us know how it goes?

**Comment by slavakurilyak** [comment]
This issue is resolved! 💃
